server/src/main/java/org/opensearch/action/bulk/BatchIngestionOption.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.bulk;
+
+import org.opensearch.common.annotation.PublicApi;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
+
+import java.io.IOException;
+
+/**
+ * An enum for batch ingestion option.
+ */
+@PublicApi(since = "2.14.0")
+public enum BatchIngestionOption implements Writeable {
+    NONE("disabled"),
```

Comment:
Curious why there's a mismatch in enum name and string here.

Reply:
`NONE` feels more like a default behavior when user doesn't do anything. The value is used to match user's input which `disabled` is a user's action.

Reply:
This *is* weird from the API POV. We're saying users can specify "batch_ingestion_ioption: enabled/disabled", but when we add "parallel" this will make no sense: it's enabled when it's parallel. 

I think the allowed options should be "none, default, parallel" and the enums should match.